vyper/parser/function_definitions/parse_public_function.py

```diff
@@ -188,7 +187,7 @@ def parse_public_function(code: ast.FunctionDef,
                 current_sig_arg_names = {x.name for x in default_sig.args}
                 base_arg_names = {arg.name for arg in sig.base_args}
                 copier_arg_count = len(default_sig.args) - len(sig.base_args)
-                copier_arg_names = current_sig_arg_names - base_arg_names
+                copier_arg_names = list(current_sig_arg_names - base_arg_names)
```

Comment:
Oh, I see now that it doesn't matter if `copier_arg_names` is sorted on this line, only after the next line (which depends on default_sig.args being ordered)

Reply:
I'd still rather have this sorted. :)